### ISSUE TITLE
Help Texts Missing

### ISSUE THREAD

**Issue by mstaack** [optimistdigital/nova-translatable#34]
Hey there,

since a week or two help texts disappeared.

```
optimistdigital/nova-translatable        1.6.1
laravel/nova                             v3.11.0
```
Any Ideas what has happend?

missing:
```
            Text::make('Link Text')->nullable()->translatable()->help(
                'Placeholder %VENUE_NAME% can be used here'
            ),
```

available:
```
            Text::make('Link Text')->nullable()->help(
                'Placeholder %VENUE_NAME% can be used here'
            ),
```

**Comment by Tarpsvo** [comment]
Hi! Thanks for reporting this, the help texts were indeed missing! Seems that Nova had added a new prop `showHelpText` that I wasn't passing along. Fixed in the latest version (1.6.3). Thanks again and good luck!

**Comment by mstaack** [comment]
brilliant!
